src/main/java/seedu/address/logic/commands/listcommand/ListIssueCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands.listcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.CommandResult;
+
+public class ListIssueCommand extends ListCommand {
+
+    /* Possible Fields */
+
+    public static final String ENTITY_TYPE = "issue";
+
+    @Override
+    public CommandResult execute(EntityList entityList) {
+        requireNonNull(entityList);
+
+        // Possible format
+        /*
+         * IssueList issueList = entityList.getIssueList();
+         * for (Issue issue : issueList) {
+         *     print(issue); // implement toString()
+         * }
+         */
+        /*
+         * entityList.getIssueList()
+         *           .list()
+         *           .stream()
+         *           .forEach(Issue::toString());
```

Comment:
the type in the forEach should be `Entity`, because `list()` returns a `List<? extends Entity>` field. But yea can count on dynamic overriding to get the correct string format.